cmd/api/main.go

```diff
@@ -4,16 +4,28 @@ import (
 	"log"
 	"net/http"
 	"os"
-    "github.com/SimplQ/simplQ-golang/internal/mux"
+
+	"github.com/SimplQ/simplQ-golang/internal/datastore"
+	"github.com/SimplQ/simplQ-golang/internal/mux"
 )
 
 func main() {
 	listenAddr := ":8080"
+
+    // Use local mongodb instance if env variable not set
+    mongoUri := "mongodb://localhost:27017/?maxPoolSize=20&w=majority"
+
 	if val, ok := os.LookupEnv("FUNCTIONS_CUSTOMHANDLER_PORT"); ok {
 		listenAddr = ":" + val
 	}
+    
+    if val, ok := os.LookupEnv("MONGO_URI"); ok {
+        mongoUri = val
+	}
```

Comment:
Let's move this and the above lines into the function `NewMongoDB()`

Later we will find a good configuration tool and move the stuff into a config file, but for now we will just keep the interfaces and layers clean.